File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C+DS/codes/Recursion/p22.c
/* Count occurence of zero in given number using recursion */
# include <stdio.h>
int czero (int num) {
	if (num==0) 
		return 0 ;
	if (num%10==0) 
		return czero(num/10)+1 ;
	if (num>0)
		return czero(num/10) ;
}
void main () {
	int num ;
	printf("Enter number\n") ;
	scanf("%d",&num) ;

	if (num==0) 
		printf("zero occures 1 time\n") ;
	else {
		int ret=czero(num) ;
		printf("zero occures %d time\n",ret) ;
	}
}

File: java-main/java-main/NestedforloopAssi/p22.java
/*
1
3 4
6 7 8
10 11 12 13
15 16 17 18 19
*/

class Demo{

	public static void main(String[]args){

		int N=4;
		int num=1;

		for(int i=1;i<=N;i++){
			for(int j=1;j<=i;j++){
				System.out.print(num++ + " ");
			}
			num++;
			System.out.println();
		}
	}

}

